indico/modules/events/papers/forms.py

```diff
@@ -88,6 +88,10 @@ class PaperReviewingSettingsForm(IndicoForm):
     announcement = IndicoMarkdownField(_('Announcement'), editor=True)
     scale_lower = IntegerField(_('Scale (from)'), [InputRequired()])
     scale_upper = IntegerField(_('Scale (to)'), [InputRequired()])
+    hide_accepted = BooleanField(_('Keep papers hidden'),
+                                 widget=SwitchWidget(),
+                                 description=_("Keep papers hidden from participants \
+                                               even after they've been accepted."))
```

Comment:
```suggestion
                                 description=_('Keep papers hidden from participants '
                                               "even after they've been accepted."))
```

backslashes are ugly :)